rotkehlchen/db/upgrades/v40_v41.py

```diff
@@ -22,15 +23,36 @@ def _add_cache_table(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_cache_table')
 
 
+def _move_non_settings_mappings_to_cache(write_cursor: 'DBCursor') -> None:
+    """Move the non-settings value from `settings` to a seperate `key_value_cache` table"""
+    log.debug('Enter _move_non_settings_mappings_to_cache')
+    settings_moved = [x.serialize() for x in DBCache]
```

Comment:
nitpick: about what I said in the other comment I believe is better to have them hardcoded since if we delete something or rename something this can fail